content/en/docs/concepts/storage/volumes.md

```diff
@@ -996,20 +996,22 @@ For more information, see the [vSphere volume](https://github.com/kubernetes/exa
 
 {{< feature-state for_k8s_version="v1.19" state="beta" >}}
 
-The `CSIMigration` feature for `vsphereVolume`, when enabled, redirects all plugin operations
-from the existing in-tree plugin to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver. In order to use this feature, the
+The `CSIMigrationvSphere` feature for `vsphereVolume` is enabled by default as of Kubernetes v1.25.
```

Comment:
Does this mean that the feature gate doc should be updated to reflect this change?

Reply:
You are right. It's covered by this other PR: https://github.com/kubernetes/website/pull/35685